lib/db/mysql.js

```diff
@@ -1535,5 +1535,43 @@ module.exports = function (log, error) {
       })
   }
 
+  const CREATE_RECOVERY_KEY = 'CALL createRecoveryKey_1(?, ?, ?)'
+  MySql.prototype.createRecoveryKey = function (uid, data) {
+    const recoveryKeyId = data.recoveryKeyId
+    const recoveryData = data.recoveryData
+    return this.write(CREATE_RECOVERY_KEY, [uid, recoveryKeyId, recoveryData])
+      .then(() => {
+        return {}
+      })
+      .catch((err) => {
+        if (err.errno === 1643) {
```

Comment:
Could you please add `const ER_SIGNAL_NOT_FOUND = 1643` at the top of this file, and replace occurrences of this magic number with the named constant?